### PR TITLE
R markdown presentation

### PR DESCRIPTION
**Pull request by luisesanmartin**

### CODE REVIEW

File: Presentations/CE-r-markdown.Rmd [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+## Exercise 4
+
+Include an R plot in your document.
+
+1. Create a new header named `R Plots` at the bottom of `markdown-template.Rmd`
+2. Create a new fenced code block with the option `echo` set to `FALSE`
+3. Add the following code inside the new block:
+
+```{r, eval = FALSE}
+plot(mtcars$wt,
```
Comment (luisesanmartin) [inline Presentations/CE-r-markdown.Rmd:364]: @luizaandrade with this plot I wanted to keep a balance between showing something that is not complicated so the data vis code syntax doesn't distract from R Markdown and including a plot not distractingly ugly, while not having to import an external library (like `ggplot2`) for it

**Comment by luisesanmartin** [comment]
@luizaandrade you'll note that I'm including more screen captures than code snippets for the text formatting and code blocks configuration parts. That's because I wanted the code to look like participants will see it in their RStudio and PDF viewer windows

File: Presentations/CE-r-markdown.Rmd [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+
+```{r echo = FALSE, out.width = "70%"}
+knitr::include_graphics("img/r-markdown/markdown-text-emphasis.png")
+```
+
+---
+
+# Markdown - Lists
+
+- Markdown allows to use ordered and unordered lists.
```
Comment (luizaandrade) [inline Presentations/CE-r-markdown.Rmd:208]:
```suggestion
- Markdown allows us to use both ordered and unordered lists.
```

File: Presentations/CE-r-markdown.Rmd [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+```{r}`r ''`
+# Your R code goes here
+```
+````
+
+---
+
+# Including R code
+
+- You can type the fenced code blocks directly or go to `Insert` > `R` in the script panel of RStudio to create a new block
```
Comment (luizaandrade) [inline Presentations/CE-r-markdown.Rmd:249]:
```suggestion
- To add new block, you can type the fences directly, go to `Insert` > `R` in the script panel of RStudio, or type `CTRL + ALT + i`
```

File: Presentations/CE-r-markdown.Rmd (hunk 1 above)
Comment (luizaandrade) [reply Presentations/CE-r-markdown.Rmd:364]: That seems fine to me. I'd mention it during the presentation, though: just say that if they have taken our R course, they probably know this is not our preferred way of making plots, but that for this session we only wanted to use base R so we don't need to get into discussing packages.

**Review by luizaandrade — APPROVED** [review]

**Comment by luizaandrade** [comment]
Note that people may run into issues if they open the PDF in Adobe and try to knit the document while it's open. I don't think we need to add this to the presentation, but you should be ready to answer this question if it comes up

**Pull request merged**